server/mdm/apple/apple_mdm.go

```diff
@@ -608,12 +603,6 @@ func (d *DEPService) processDeviceResponse(ctx context.Context, depClient *godep
 		}
 
 		for orgName, serials := range assignSerials {
-			decTok, err := assets.ABMToken(ctx, d.ds, orgName)
-			if err != nil {
-				return ctxerr.Wrap(ctx, err, "decrypting ABM token")
-			}
-			ctx = abmctx.NewContext(ctx, decTok)
```

Comment:
Note that I removed those because for the ABM terms expired check to work, it's important that we rely on the name of the org only, that the raw token is NOT passed in the context (this should only be used when using a token that is not yet saved to the DB).

Reply:
@mna yeah good call, I forgot to remove this in my PR after it was decided that we'd stick with looking up existing tokens in the DB.